cmd/logcli/main.go

```diff
@@ -138,6 +141,33 @@ func main() {
 		}()
 	}
 
+	if *stdin {
+		queryClient = client.NewFileClient(os.Stdin)
```

Comment:
Well, if you support this could we also support `file://my/location/text.log`?

Reply:
We could, but the same thing can be done via `cat <file> | logcli query ...`. Plus that may introduce new flag?. 
wondering do you have any other usecase for this?

Reply:
Just for ease of use. This could be a follow up though.